### PR TITLE
Push User Management to Production

### PR DESCRIPTION
**Pull request by MaxWardle**
See changes in Development here: https://dev.developer.gov.bc.ca/docs/default/component/public-cloud-techdocs/user-management/

### CODE REVIEW

**Review by prabhukiran9999 — APPROVED** [review]

**Review by wrnu — APPROVED** [review]

**Review by Pilargit12 — COMMENTED** [review]
Hi Team, could we please make sure that any image follows alt ext for accessibility purposes please? 

I believe the alt text could be added by modifying the name of the  .png with a more descriptive title. 

For example: 

Instead of: langing_zone.png
Use: aws landing zone screen preview of the dashboard showing access to choose different types to login based on user credentials.png 

Instead of: registry_roles.png
Use: aws landing zone screen preview of the dashboard to manage users showing different users with different access options.png 

For more information:

https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit/accessible-digital-content/graphics#alternative-text

**Alternative text**
Alternative text, also known as alt text, describes the information or function of digital images. Alt text should be clear and concise, try to stay under 15 words. If you're struggling to describe it, think how you would describe it to someone over the phone.

Do not include 'image of' or 'photo of' in the alt text, screen readers will announce it as an image.

File: docs/user-management.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+One of the key updates is the introduction of a comprehensive user management feature. This feature allows product owners and technical leads to manage AWS permissions for users associated with their product. Users can be assigned to one or more of the following roles:
+
+1. **Viewers**
+2. **Billing viewers**
+3. **Developers**
+4. **Admins**
+5. **Security auditors**
+
+![New role management feature in the product registry](images/user-management/registry_roles.png)
```
Comment (MaxWardle) [inline docs/user-management.md:20]: @Pilargit12 Hi Pilar, the text in the square brackets is actually the image alt text, not the file name. Would you like me to change what is there currently based on your suggestions?

Comment (Pilargit12) [reply docs/user-management.md:20]: You are right I had missed that sorry!

Comment (MaxWardle) [reply docs/user-management.md:20]: Just going to close this, change the alt text and create a new PR dev -> main

**Pull request closed without merging**